editor.html

```diff
@@ -167,7 +167,7 @@ <h2><i class="fa fa-unlock-alt"></i> <strong>{{ title }}</strong></h2>
             <p>{{ hint }}</p>
             <p><button id="button-decrypt-link">{{ button }}</button></p>
         </script>
-        <div id="main" class="vbox">
+        <main id="main" class="vbox">
```

Comment:
Not 100% sure how this is supported in IE10+ (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main), so we should test it with a VM before merging.

Reply:
We should probably also add the role="main" attribute.

Reply:
I've tested the code with <main> in ie10 on win7 and it doesn't appear to cause an issue for normal use. We could use role="main" as well/instead to cover iE10 usage.

Reply:
Fantastic, thanks!

Reply:
I think something went wrong and removed the `role="main"` attribute?
Maybe a force-push? Check if there are any other regressions.

Reply:
`role="main"` is still missing here.

Reply:
Not sure why that happened, i think i may have removed an extra commit when changing the aria-label stuff